src/main/java/com/github/soerxpso/xpspawners/manager/ConfigManager.java

```diff
@@ -0,0 +1,22 @@
+package com.github.soerxpso.xpspawners.manager;
+
+import org.bukkit.configuration.file.FileConfiguration;
+
+import com.github.soerxpso.xpspawners.XPSpawners;
+
+public class ConfigManager {
+
+	private static FileConfiguration config;
+	
+	public static void loadConfig() {
+		config = XPSpawners.getPlugin().getConfig();
```

Comment:
I usually use this pattern:

```
XPSpawners.getPlugin().saveDefaultConfig();
XPSpawners.getPlugin().reloadConfig();
```
followed by the code that you have

Note this only saves the default if no config exists on the file system